docs/sdk/wrapped-keys/overview.md

```diff
@@ -60,12 +40,146 @@ yarn add @lit-protocol/wrapped-keys
+5. The SDK returns a [ImportPrivateKeyResult](https://v6-api-doc-lit-js-sdk.vercel.app/interfaces/wrapped_keys_src.ImportPrivateKeyResult.html) object containing the generated Wrapped Key ID, and the PKP Ethereum address the Wrapped Key is associated with
+
+### Exporting a Wrapped Key
+  
+1. The Wrapped Keys SDK will use the provided Wrapped Key ID and PKP Session Signatures to fetch the encryption metadata for a specific Wrapped Key
+2. Using the PKP Session Signatures, the SDK will make a request to the Lit network to execute the [exportPrivateKey](https://github.com/LIT-Protocol/js-sdk/blob/master/packages/wrapped-keys/src/lib/litActions/common/src/exportPrivateKey.js) Lit Action
+3. The Lit Action will check the Access Control Conditions the plaintext private key was encrypted with to verify the PKP is authorized to decrypt the private key
+4. If authorized, the unencrypted plaintext private key will be returned. If not authorized, an error will be returned
+
+### Getting Wrapped Key Metadata
+
+1. The Wrapped Keys SDK will use the provided Wrapped Key ID and PKP Session Signatures to fetch the stored metadata for a specific Wrapped Key from the Wrapped Keys backend service
+2. The stored metadata is returned as a [StoredKeyData](https://v6-api-doc-lit-js-sdk.vercel.app/interfaces/wrapped_keys_src.StoredKeyData.html) object
+
+### Storing Wrapped Key Metadata
+
+:::note
+The [StoreEncryptedKeyParams](https://v6-api-doc-lit-js-sdk.vercel.app/types/wrapped_keys_src.StoreEncryptedKeyParams.html) required for this method include the encryption metadata of the private key that will be turned into a Wrapped Key. For more information on how to obtain the encryption metadata, please refer to this guide on [encrypting data using the Lit SDK](../../sdk/wrapped-keys/custom-wrapped-keys#generating-and-encrypting-a-private-key), and this guide on [Custom Wrapped Keys](http://localhost:3000/sdk/wrapped-keys/custom-wrapped-keys#generating-and-encrypting-a-private-key).
+:::
+
+1. The Wrapped Keys SDK will derive the PKP's Ethereum address from the provided PKP Session Signatures
+2. The SDK stores the provided [StoreEncryptedKeyParams](https://v6-api-doc-lit-js-sdk.vercel.app/types/wrapped_keys_src.StoreEncryptedKeyParams.html) using the Wrapped Keys backend service, associating the resulting Wrapped Key with the PKP's Ethereum address
+3. The SDK returns a [StoreEncryptedKeyResult](https://v6-api-doc-lit-js-sdk.vercel.app/interfaces/wrapped_keys_src.StoreEncryptedKeyResult.html) object containing the generated Wrapped Key ID, and the PKP Ethereum address the Wrapped Key is associated with
+
+### Listing Wrapped Keys for a PKP   
+   
+1.  The Wrapped Keys SDK will derive the PKP Ethereum address from the provided PKP Session Signatures
+2.  The SDK will submit the Ethereum address to the Wrapped Keys backend service to fetch and return all the associated Wrapped Keys
+
+### Signing an Arbitrary Message
```

Comment:
it may be helpful to add a few sentences distinguishing generic 'message' signing vs 'transaction' signing

Reply:
like when one would use one over the other

Reply:
Added

```
- [Signing an arbitrary message/data with a Wrapped Key](./sign-message.md)
  - Useful for non transactional data such as:
    - Creating a digital signature of data to verify it hasn't been tampered with
    - Proving ownership (authenticating) of a specific blockchain address
    - Creating and verifying credentials or attestations in decentralized identity systems
```